determined-ai/determined
# feat: replace custom `isEqual` to lodash `isEqual`

## Description
[WEB-1086]




## Test Plan
- Check if there's no custom `isEqual` is used
- Check if there's no regression
- Check if CI passes




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1086]: https://hpe-aiatscale.atlassian.net/browse/WEB-1086?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
